pulsar-common/src/main/java/org/apache/pulsar/common/util/collections/ConcurrentLongPairSet.java

```diff
@@ -353,6 +357,8 @@ boolean contains(long item1, long item2, int hash) {
                             stamp = readLock();
                             acquiredLock = true;
 
+                            // update local variable
+                            table = this.table;
                             bucket = signSafeMod(hash, capacity);
```

Comment:
similar comment: why use capacity instead of `table.length / 2`?

Reply:
fixed.